docs/uds-packages/start_here.md

```diff
@@ -0,0 +1,40 @@
+# Quick Start - UDS Packages
+
+This Quick Start is intended to provide the most concise and direct path to creating a Bronze UDS package.
+
+> [!TIP] Why?
+> Defense Unicorns is focused on making software conveniently available to the warfighter. _wherever the mission may take them_.
+>
+> Creating a UDS package makes that piece of software compatible and available in the UDS ecosystem.
+
+<p align="center">
+    <img align="top" src=".images/boromir.png" alt="alt text" width="200">
+    <img src=".images/wat-no-not-that-wat.png" alt="alt text" width="200">
+</p>
+
+1. `git clone https://github.com/defenseunicorns/uds-package-template.git` find and replace variables
+2. `zarf dev generate -h` to generate a `zarf.yaml` file from a helm chart
+   1. Replace the `common/zarf.yaml` with the generated
+3. Update the `zarf.yaml` (top-level)  with the correct reference. Add the required images to the `upstream` flavor
```

Comment:
correct reference to?

Reply:
the common import should be wired up - should this be metadata?

Reply:
you are right. made some updates